### PR TITLE
[#84] Beautify -h (4×' ', 15×opt, desc)

### PR DESCRIPTION
**Pull request by pickfire**
The revive of the fallen.

### CODE REVIEW

File: functions/fisher_help.fish [hunk 1]
```diff
@@ -29,16 +29,12 @@ function fisher_help -d "Show Help"
...
-
             case h
                 printf "Usage: fisher help [<keyword>] [--all] [--guides] [--help]\n\n"
-
-                printf "           -a --all  List available documentation\n"
-                printf "        -g --guides  List available guides\n"
+                printf "    -a --all       List available documentation\n"
+                printf "    -g --guides    List available guides\n"
+                printf "    -h --help      Show usage help\n"
                 printf "    --usage[=<cmd>]  Display command usage\n"
```
Comment (bucaran) [inline functions/fisher_help.fish]: Maybe align `Show usage help` with `List available...`?

**Comment by bucaran** [comment]
Thanks @pickfire!

File: functions/fisher_help.fish (hunk 1 above)
Comment (pickfire) [inline functions/fisher_help.fish]: What do you mean?

**Comment by pickfire** [comment]
@bucaran I am currently working on updating the man pages, is there any way I could test it? How do I make those man files? `make man`?

**Comment by bucaran** [comment]
@pickfire Before you do any work, let's create an issue to track what changes are required using beautiful check boxes.

+ [ ] Check me!

**Comment by pickfire** [comment]
- [ ] Beautify those ascii tree diagram
- [ ] [Words to avoid](https://github.com/nicholaides/words-to-avoid.vim) in those articles
- [ ] Misc fixes / forgotten updates

**Comment by bucaran** [comment]
Not here. Create a new issue for this.

**Comment by pickfire** [comment]
No problem. #129

**Pull request merged**